test/sharness/t0119-prometheus.sh

```diff
@@ -8,7 +8,7 @@ test_description="Test prometheus metrics are exposed correctly"
 
 . lib/test-lib.sh
 
-test_init_ipfs
+test_init_ipfs_measure
```

Comment:
nit: this no longer tests metrics exposed by Kubo default `ipfs init`

@gammazero mind updating this?

- use the `test_init_ipfs` and update `t0119-prometheus-data/prometheus_metrics` and `t0119-prometheus-data/prometheus_metrics_added_by_enabling_rcmgr` to remove extra metrics
- add dedicated test for `ipfs init` with `flatfs-measure` profile, similar to https://github.com/ipfs/kubo/blob/fc44f117832d9f3aa73f3e7a055ce62f7bf109da/test/sharness/t0119-prometheus.sh#L56C1-L58C2  (confirming extra metrics are registered)

Reply:
Done